README.md

```diff
@@ -642,7 +642,7 @@ Committers can later re-run this license-check workflow from the Github actions
 #### Requirements
 - Maven based build
 - Root pom.xml must reside in the repository root
-- An [authentication token (scope: api) from gitlab.eclipse.org](README.md#automatic-ip-team-review-requests) has to be stored in the repositories secret store(Settings -> Scrects -> Actions) with name `M2E_GITLAB_API_TOKEN`.
+- An [authentication token (scope: api) from gitlab.eclipse.org](README.md#automatic-ip-team-review-requests) has to be stored in the repositories secret store(Settings -> Scrects -> Actions) with name `<PROJECT-NAME>_GITLAB_API_TOKEN`, such that it can be used by the above workflow definition.
```

Comment:
Could this opportunity be used to fix `-> Scrects ->` to probably `-> Secrets ->`? And perhaps to add space after `secret store`.

Reply:
This still doesn't look right:
![image](https://github.com/user-attachments/assets/f791b279-c27f-4684-8627-e80159ea9968)

Reply:
Thanks, fixed.